lib/ecto_ldap/adapter.ex

```diff
@@ -4,6 +4,8 @@ defmodule Ecto.Ldap.Adapter do
 
   @behaviour Ecto.Adapter
 
+  @server_timeout 5000
```

Comment:
Yep!  This is actually intended for a convenience when stubbing in `:infinity` when debugging via `IEx`.  When debugging `GenServer`, I was hitting timeouts.  This avoids having to replace the value on all the fields.